diff --git a/node_modules/@types/mongodb/index.d.ts b/node_modules/@types/mongodb/index.d.ts
index d37f518..c07a5ae 100755
--- a/node_modules/@types/mongodb/index.d.ts
+++ b/node_modules/@types/mongodb/index.d.ts
@@ -55,7 +55,7 @@ export function connect(uri: string, options?: MongoClientOptions): Promise<Mong
 export function connect(uri: string, callback: MongoCallback<MongoClient>): void;
 export function connect(uri: string, options: MongoClientOptions, callback: MongoCallback<MongoClient>): void;
 
-export { Binary, DBRef, Decimal128, Double, Int32, Long, MaxKey, MinKey, ObjectID, ObjectId, Timestamp } from "bson";
+export { Binary, DBRef, Decimal128, Double, Int32, Long, MaxKey, MinKey, ObjectId, ObjectId as ObjectID, Timestamp } from "bson";
 
 type NumericTypes = number | Decimal128 | Double | Int32 | Long;
 
